### DISCUSSION TITLE
Vocês tem sites/blogs?

### DISCUSSION THREAD

**Discussion by raphaklaus** [frontendbr/forum#91] [Discussão]
E aí pessoal!

Vocês tem sites/blogs onde postam suas descobertas/conhecimentos/projetos?

O meu é: [raphaklaus.com](http://raphaklaus.com)

**Comment by danguilherme** [comment]
Opa! Legal hein, muito bom o site!

Estou começando agora,  não tem quase nada no site ainda e ainda tô pra comprar um domínio, não decidi o nome :/

O site é https://danguilherme.github.io/.

Pior que bem lá atrás eu tinha criado um site "placeholder" mas não gostei do domínio... Hahaha - é o http://guilhermeventura.net/. Um dia desses junto os dois.

**Comment by heliomsolivas** [comment]
Eu também fiz um não faz muito tempo, mas não consigo tempo pra escrever :(

https://heliomsolivas.github.io

**Comment by renanvaz** [comment]
Cara eu já tentei ter um blog, mas não consegui ficar ativo e desisti.
O meu site é antigo (estrutura), mas não tenho pretensão de atualiza-lo por enquanto.
http://renanvaz.com.br/

**Comment by brunoravanhani** [comment]
Eu to meio que tentando produzir um conteúdo e ainda terminar o site :) mas eu frequentemente posto uma serie de links toda semana, mas to tentando achar um bom formato ainda, mas ta ai: 
http://ravanhani.com/

**Comment by brunoravanhani** [comment]
@renanvaz que background legal nesse teu side ai cara kkk. Massa mesmo

**Comment by lucasbento** [comment]
Estou tentando criar coragem à tempos para criar um site/portfólio e ainda não consegui.

---

@renanvaz muito legal o background mesmo!
@raphaklaus curti o background do menu, deu um efeito bem legal.

**Comment by dleitee** [comment]
@renanvaz como que tu fez esse background?

**Comment by renanvaz** [comment]
@dleitee No background eu fiz um gráfico de voronoi com o D3js.
O "joguinho" com o ETE eu fiz como teste de uma engine de física que fiz na época.

**Comment by dleitee** [comment]
@renanvaz muito massa! Parabéns!!

**Comment by raphaklaus** [comment]
@lucasbento Heheh valeu!!

@renanvaz Eu pensei algo parecido pro meu! Mas talvez faça algo assim num futuro indefinido. :stuck_out_tongue:

**Comment by renanvaz** [comment]
Que legal que gostaram pessoal, valeu!

@raphaklaus o gráfico de voronoi é legal, mas ele é bem pesado em questão de processamento. Se vc não precisar de interação de mouse (de fábrica) uma opção legal é o [trianglify](https://github.com/qrohlf/trianglify)

**Comment by diessica** [comment]
Tenho um blog. :)

https://diessi.ca

**Comment by kazzkiq** [comment]
Salve @diessica! Foi graças a uma [resposta sua lá no Stack Overflow PT](http://pt.stackoverflow.com/a/2939/1754) que eu aprendi a técnica de centralizar elementos usando `translate()`! De lá pra cá já usei DEMAIS essa técnica em vários projetos (provavelmente to passando vergonha agora porque todo mundo já devia saber usar `translate()` em 2014, mas tudo bem :smile:).

Enfim, só queria aproveitar a oportunidade pra agradecer pelo ensinamento mesmo (lá no SO o ambiente é mais _formalzão_ então não da pra ficar escrevendo comments do tipo: "brigado", "valeu!!11!", etc).

**Comment by willianjusten** [comment]
Eu tenho um blog tb =)

http://willianjusten.com.br/

**Comment by robsonbittencourt** [comment]
Não falo especificamente de front, mas tenho um blog também:

[rbittencourt.com](rbittencourt.com)

**Comment by felipefialho** [comment]
Infelizmente está praticamente abandonado por problemas na versão do Docpad que estou usando, mas vamos ver se consigo consertar isso em breve...

http://www.felipefialho.com/

**Comment by wellingtonpgp** [comment]
Tenho site é blog, mas faz tempo que tempo que não mexo no blog, depois que crie a página Diário de um Back-End, depois da um like lá galera @willianjusten aprova Diário de um Back-End 
Link da página: https://www.facebook.com/diariobackend
Meu site: http://wellingtongeek.com/ e meu blog que faz tempo que não compartilho nada lá http://blog.wellingtongeek.com :+1:

**Comment by willianjusten** [comment]
Aprovo o Diário de um Back-End sim, só acho até hoje que deveria se chamar Diário de um Desenvolvedor =p

**Comment by wellingtonpgp** [comment]
@willianjusten , não mudei ainda pois não deu tempo de bolar uma nota logo, mas deixei uma nota que vejo e lembro todo dia para mudar =P

**Comment by diessica** [comment]
@kazzkiq, gente! Que lindo! Muito obrigada! Fico imensuravelmente feliz de ter te ajudado, de verdade!

**Comment by baltazarparra** [comment]
http://baltazarparra.github.io/

Jekyll, ou quase. Estou escrevendo uma serie sobre flexbox, depois vou começar uma sobre CSS Grid Layout.

**Comment by rickbenetti** [comment]
Eu to reescrevendo meu site em jekyll pra ser meu portifa e outras coisas (https://rkb81.com.br), por isso to repassando os poucos posts interessantes pro medium.com, buscando outra abordagem ao invés de compartilhar códigos, estou compartilhando minha experiência: https://medium.com/wdsign-brasil/

**Comment by wbruno** [comment]
**blog**
http://wbruno.com.br

**Comment by woliveiras** [comment]
**Blog**

http://woliveiras.com.br/

Está parado ultimamente devido ao ritmo frenético de estudos! ;)

**Comment by baltazarparra** [comment]
@woliveiras eu acompanhava seu blog, deveria fazer um esforço e voltar a escrever.

**Comment by woliveiras** [comment]
@baltazarparra, valeu pelo incentivo cara. Eu finalizei esses dias a migração dos posts que ainda não estavam revisados (naquela migração de WordPress pra Hexo). Logo logo voltarei a postar sim. Ainda mais que já aprendi muita coisa nova! :smile:

**Comment by brunopulis** [comment]
Tem um tempo que nao posto mas ta ai [http://brunopulis.com/blog/](http://brunopulis.com/blog/)

**Comment by doug2k1** [comment]
Migrei o meu de Wordpress para Jekyll, mas falta postar mais:
http://dmatoso.com

**Comment by endersoneac** [comment]
Criei um recentemente.
http://endersoncosta.com/blog

ainda não coloquei tudo lá, mas logo logo finalizo ele.

**Comment by lagden** [comment]
Sim, só que não(sqn) escrevo faz tempo!

http://lagden.in
http://lagden.github.io/

o da @diessica é totalmente 🦄!

**Comment by diessica** [comment]
Que agonia ver esse 🦄 e não saber o que você quis dizer! hahahaha
<img width="294" alt="screen shot 2016-04-12 at 02 48 08" src="https://cloud.githubusercontent.com/assets/5303585/14450941/01407184-0059-11e6-80a1-7839ad2ae281.png">

**Comment by lagden** [comment]
@diessica 
unicórnio = mágico, mitológico, fabuloso, :godmode:, etc...!!
Serei mais claro na próxima!!! hihihi...

**Comment by bernardocs** [comment]
Eu tenho um **site**... [bernardocs.github.io](http://bernardocs.github.io/)

A idéia, na verdade, é ser apenas um "business card", só pra galera conseguir me encontrar wherever.

**Comment by teraboxpremiummodapk-svg** [comment]
That’s awesome, thanks for sharing your blog! I’ve also been documenting my projects and tips, and one tool that’s made a huge difference for me is [TeraBox Premium MOD APK](https://ultraterabox.com/). With the unlimited cloud storage it offers, I can back up all my project files, share them easily, and keep everything synced across devices. It’s been a real game-changer for managing resources while working on different projects.

**Comment by EliasOPrado** [comment]
Tenho um que falo sobre tecnologia e posicionamento profissional.

[prosimples.com](https://prosimples.com)

<img width="33%" height="1667" alt="image" src="https://github.com/user-attachments/assets/e445aa77-38b9-44bf-80c7-d0b03f4fa63d" />


Talvez te ajude.
